### PR TITLE
Fix missing docstring

### PR DESCRIPTION
**Pull request by TobKed**

### CODE REVIEW

File: airflow_munchkin/client_parser/cloud_client_parser.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ def parse_path_method(name: str, method: Callable) -> PathInfo:
 
 def parse_action_method(name: str, method: Callable) -> ActionInfo:
     docstring = method.__doc__
-    assert docstring is not None
+    if docstring is None:
+        docstring = """None # TODO: Fill missing value"""
```
Comment (nuclearpinguin) [inline airflow_munchkin/client_parser/cloud_client_parser.py]: Just a question, do we have to use triple quote?

Comment (TobKed) [reply airflow_munchkin/client_parser/cloud_client_parser.py]: We don't have to but it seems a good choice due to **PEP 257**:  https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

> Triple quotes are used even though the string fits on one line. This makes it easy to later expand it.

**Comment by TobKed** [comment]
cc @mik-laj @potiuk @nuclearpinguin

File: airflow_munchkin/client_parser/cloud_client_parser.py (hunk 1 above)
Comment (potiuk) [reply airflow_munchkin/client_parser/cloud_client_parser.py]: But in PEP 257 it's only about "real" docstrings. If I am not mistaken this is just a "string" after it gets assigned to the docstring variable and it makes no difference if you have """ """ it's more how you generate the code - eventually the docstring will have """ anyway I guess.

Comment (TobKed) [reply airflow_munchkin/client_parser/cloud_client_parser.py]: Good point @potiuk. I've fixed it.

**Comment by nuclearpinguin** [comment]
@TobKed can you fix the mypy error?

**Comment by TobKed** [comment]
Rebased and test added.

**Pull request merged**